concepts/callbacks/.meta/config.json

```diff
@@ -1,5 +1,5 @@
 {
-  "blurb": "TODO: add blurb for callbacks concept",
+  "blurb": "A callback is a functions passed as an argument to another function which invokes at the end of its execution",
```

Comment:
It's not necessarily at the end of the execution, and not necessarily invoked. Maybe a way to capture that too within the character limit?